releases/development/master/extra/install-chef-suse.sh

```diff
@@ -679,13 +679,20 @@ if [ -n "$CROWBAR_FROM_GIT" ]; then
     fi
 
     # install chef and its dependencies
-    zypper -n --gpg-auto-import-keys in rubygem-chef-server rubygem-chef rabbitmq-server \
-            couchdb rubygem-activesupport
+    zypper -n --gpg-auto-import-keys in ruby2.1-rubygem-chef-server ruby2.1-rubygem-chef \
+        rabbitmq-server couchdb ruby2.1-rubygem-activesupport
 
     # also need these (crowbar dependencies):
-    zypper -n in rubygem-app_config rubygem-cstruct rubygem-kwalify rubygem-ruby-shadow \
-            rubygem-sass rubygem-i18n sleshammer tcpdump rubygem-sprockets-helpers \
-            rubygem-sprockets-sass rubygem-sqlite3
+    zypper -n in \
+        ruby2.1-rubygem-cstruct \
```

Comment:
In these ruby gem packages are no patches so we dont need to install it manually?